src/installer.ts

```diff
@@ -42,31 +39,62 @@ export async function getNode(
     console.log(`Found in cache @ ${toolPath}`);
   } else {
     console.log(`Attempting to download ${versionSpec}...`);
-    let info = await getInfoFromManifest(versionSpec, stable, token);
-    if (!info) {
-      console.log(
-        'Not found in manifest.  Falling back to download directly from Node'
-      );
-      info = await getInfoFromDist(versionSpec);
-    }
-
-    if (!info) {
-      throw new Error(
-        `Unable to find Node version '${versionSpec}' for platform ${osPlat} and architecture ${osArch}.`
-      );
-    }
-
-    console.log(`Acquiring ${info.resolvedVersion} from ${info.downloadUrl}`);
-
     let downloadPath = '';
+    let info: INodeVersionInfo | null = null;
+
+    //
+    // Try download from internal distribution (popular versions only)
```

Comment:
get info + download inside try/catch

The formatting rules bloats the line count quite a bit. We might be on old formatting rules because lots of extra newlines added by formatting rules.